float64 mobSumUpdateInterval
float64 speedDriftThreshold